### ISSUE TITLE
[Topic] Introduction to WCAG

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#92]
URL: https://wpaccessibility.org/docs/start/wcag-intro/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/start/wcag-intro/index.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

- Review current content, keep it readable for a large audience
- Add information about versions and the meaning status Recommended version
- Add info about difference between principle/ guideline/succes criteria/technique. This to avoid people will confuse a technique for a succes criterion.
- Link to Standards and best practice for more in dept info

## Discussion

What is the baseline now for WP and for worldwide accessibility testing? 
Like: Browser versions and browser/AT combinations.
Is that written down with the new accessibility ready theme requirements?
Do we need to add this baseline here or on a different page?

Comments from @AnneliesDT about the current content:
> I’m missing the relationship between EN 301549 and WCAG, and the relationship between the different laws and WCAG. Even if it’s just short, it’s good to mention that the WAD and EAA and others elsewhere, use the EN 301549 as a basis, which in turn uses WCAG as a basis (because it is the basis…). I would also like to see the ISO norm for accessibility mentioned.
